sbin/cubeit

```diff
@@ -173,6 +178,94 @@ if [ -n "$ARTIFACTS_DIR" ]; then
 fi
 export ARTIFACTS_DIR
 
+# determine the location of key store
+if [ -z "$KEYS_DIR" -o ! -d "$KEYS_DIR" ]; then
```

Comment:
This is should be consistent with the rest of the conditionals in the file.

if [ -z "${KEYS_DIR}" ] || [ ....